man/unshield.1

```diff
@@ -1,8 +1,8 @@
-.TH UNSHIELD "1" "July 2004" "The SynCE project" "http://www.synce.org"
+.TH UNSHIELD "1" "April 2022" "The SynCE project" "http://www.synce.org"
 .SH NAME
 unshield \- extract CAB files from an InstallShield installer archive
 .SH SYNOPSIS
-unshield [\-c COMPONENT] [\-d DIRECTORY] [\-D LEVEL] [\-g GROUP] [\-h] [\-i VERSION] [\-j] [\-L] [\-n] [\-o] [\-O] [\-r] [\-v] [\-V] c|g|l|t|x CABFILE
+unshield [\-c COMPONENT] [\-d DIRECTORY] [\-D LEVEL] [\-e ENCODING] [\-g GROUP] [\-h] [\-i VERSION] [\-j] [\-L] [\-n] [\-o] [\-O] [\-r] [\-R] [\-v] [\-V] c|g|l|t|x CABFILE
```

Comment:
Does it make sense to keep this in sync with the help output?

`unshield [-c COMPONENT] [-d DIRECTORY] [-D LEVEL] [-g GROUP] [-i VERSION] [-e ENCODING] [-GhlOrV] c|g|l|t|x CABFILE [FILENAME...]`

The order of the options is slightly different.

Reply:
I noticed this (temporary?) deactivation
https://github.com/twogood/unshield/blob/4852238d1d9aa736b48f0e7405c19ca6a0996b35/src/unshield.c#L223

Reply:
Good catch, the code is still supported (https://github.com/twogood/unshield/blob/4852238d1d9aa736b48f0e7405c19ca6a0996b35/src/unshield.c#L242 except *v*), not sure why it was hidden from the help output.

Reply:
@kratz00 As David said, it's a dummy waiting for implementation.